addOns/encoder/src/main/javahelp/org/zaproxy/addon/encoder/resources/help/contents/encoder.html

```diff
@@ -198,14 +198,23 @@ <H4>To Lower Case</H4>
 Converts the input to all lower case characters.
 
 <H4>Remove Whitespace</H4>
-Removes all whitespace characters from the text, based on <a href="https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Character.html#isWhitespace(char)">Character.isWhiteSpace(char)</a>.
+Removes all whitespace characters from the text, based on <a href="https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Character.html#isWhitespace(char)">Character.isWhiteSpace(char)</a>.
 
 <H4>Reverse</H4>
 Reverses the order of the input.
 
 <H4>To Upper Case</H4>
 Converts the input to all upper case characters.
 
+<H4>Transliterate</H4>
+Converts text removing accents/diacritics/ligatures (perhaps not fully, due to operation in compatibility mode) leaving only ASCII characters.
+Ex: <code>Tĥïŝ ĩš â fůňķŷ Šťŕĭńġ: ﬁ. étrange.</code> becomes <code>This is a funky String: fi. etrange.</code>.>br>
```

Comment:
Would be good to have an example that would show that it also strips non-ASCII characters, possibly the whole string.